src/refGraph.ts

```diff
@@ -0,0 +1,80 @@
+import { DepGraph } from 'dependency-graph';
+import * as treeify from 'treeify';
+
+export interface INode<T> {
+  id: string;
+  data?: T;
+}
+
+export interface INodeTree<T> {
+  node: INode<T>;
+  children: { [id: string]: INodeTree<T> };
+}
+
+export class RefGraph<T> extends DepGraph<T> {
+  /**
+   * id of node
+   * @param {string} id
+   */
+  public nodeTree(id: string): INodeTree<T> {
+    return nodeTree(this, { id });
+  }
+  /**
+   * id of node
+   * @param {string} id
+   */
+  public nodeChildren(id: string): Array<INode<T>> {
+    return nodeChildren(this, { id });
+  }
+
+  public serialize(id: string): string {
+    return treeify.asTree(serialize(this, { id }), false, false);
+  }
+}
+
+export const nodeChildren = <T>(graph: RefGraph<T>, node: INode<T>): Array<INode<T>> => {
+  // @ts-ignore
```

Comment:
```suggestion
```
:P

Reply:
Ahh, that one has to be there, because `DepGraph` doesn't expose that in their typescript interface.